Intel_SighPP_CMake/Intel_SighPP_CMake/src/ml_lib/yolo_impl.cpp

```diff
@@ -17,9 +17,9 @@ struct YoloModelImpl : public ModelInterface {
 
 	cv::dnn::Net net;
 	vector<string> class_names;
+	size_t inWidth;
+	size_t inHeight;	
 	
-	const size_t inWidth = 640;
-	const size_t inHeight = 480;
 	const float WHRatio = inWidth / (float)inHeight;
 	const float inScaleFactor = 0.007843f;
 	const float meanVal = 127.5;
```

Comment:
I think these variables could be put into the interface? They are the same in both caffe and yolo?